.github/workflows/sync-release-assets.yml

```diff
@@ -13,38 +13,10 @@ jobs:
   sync-github-and-dist-ipfs-io:
     runs-on: "ubuntu-latest"
     steps:
-      - name: Setup go
-        uses: actions/setup-go@v2
+      - uses: ipfs/download-ipfs-distribution-action@v1
+      - uses: ipfs/start-ipfs-daemon-action@v1
         with:
-          go-version: '1.16'
-      - uses: actions/cache@v2
-        with:
-          path: |
-            ~/.cache/go-build
-            ~/go/pkg/mod
-          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
-          restore-keys: |
-            ${{ runner.os }}-go-
-      - name: Build go-ipfs binary
-        run: go install github.com/ipfs/go-ipfs/cmd/ipfs@latest
-      - name: Initialize go-ipfs and start daemon
-        run: |
-          sudo sysctl -w net.core.rmem_max=2500000
-          ipfs init --profile flatfs,server
-          ipfs daemon --enable-gc=false &
-          while (! ipfs id --api "/ip4/127.0.0.1/tcp/5001"); do sleep 1; done
-      - name: Wait for go-ipfs to be ready
```

Comment:
@galargh Can we update `ipfs/start-ipfs-daemon-action` to include a version of this additional check?
iiuc we want to ensure the  `ipfs id | jq .Addresses`  list is not empty before continuing. I believe we had CI randomly fail due to race condition sometimes.

Reply:
Sure! https://github.com/ipfs/start-ipfs-daemon-action/issues/5